pages/mesosphere/dcos/2.0/release-notes/2.0.6/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 2.0.6
+title: Release notes for 2.0.6
+menuWeight: 0
+excerpt: Release notes for DC/OS 2.0.6, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 2.0.6 was released on ????, 2020.
+
+[button color="light" href="https://downloads.dcos.io/dcos/stable/2.0.6/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="purple" href="https://downloads.mesosphere.com/dcos-enterprise/stable/2.0.6/dcos_generate_config.ee.sh"]Download DC/OS Enterprise* [/button]
+
+New customers, contact your sales representative or <a href="mailto:sales@mesosphere.io">sales@mesosphere.io</a> before attempting to download and install DC/OS Enterprise.
+
+# Release Summary
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment.
+
+# DC/OS 
+
+## Components
+
+DC/OS 2.0.6 includes the following component versions:
+
+- ????
+
+### DC/OS Fixed and Improved Issues
```

Comment:
We are missing the most important enhancement (from the 2.0.6 changes.md file mentioned above):

"Starting services on clusters with static masters now only requires a majority of ZooKeeper nodes to be available. Previously, all ZooKeeper nodes needed to be available. On clusters with dynamic master lists, all ZooKeeper nodes must still be available. (D2IQ-4248)"

Also, this should also mention COPS-1717, which was the original COPS about this issue (from 2015!!!!!!)

This is a major improvement and has been a point of contention at several customers, so we want to make sure we have this in the Release notes so that is clear what release the fix is in.

Reply:
hey @cneth -  good catch!  we are trying really hard to catch all this information in jira, which is our "single point of truth."   

casie did capture [d2iq-4248](https://jira.d2iq.com/browse/D2IQ-4248) in the notes, in the final bulleted point.  d2iq-4248 resolves [cops-1754](https://jira.d2iq.com/browse/COPS-1754), which was reported by harpreet in 2017.  both of those issues have the same title as you can see (Master node should be able to rejoin the cluster after failure/restart when another master is offline or being upgraded).   it appears that within the jira ticket, the true benefit of D2iQ-4248 wasn't spelled out as clearly as it was in the changes.md.  

i'm commenting here because i want to improve my reporting on the jira tickets and maybe partner with you and your peers to make sure we don't miss these important points and are able to harness your excellent historical knowledge  (a shout out to you for tying in cops-1717.  i would not have thought to mention a ticket that was resolved in 1.7)!!  i'll reach out to you offline in the hopes we can work together to improve this process.  thank you!

Reply:
Yeah, I think COPS-1754 is really the right ticket, I messed up on that reference (1717 is related but not really the same exact issue).  Just wanted to make sure we did not miss mentioning this one.  I suspect there is at least one renewal held up on this point so we need to make sure it's clear this fix has been delivered.

Reply:
Ok, left description as-is with COPS 1754 and D2iq 4248.